File: 백준/Bronze/10810. 공 넣기/공 넣기.py
import sys

n, m = map(int, sys.stdin.readline().split())

list = []

for i in range(n + 1):
    list.append(0)

for i in range(m):
    i, j, k = map(int, sys.stdin.readline().split())
    for a in range(i, j + 1):
        list[a] = k

print(*list[1:])

File: 백준/Bronze/10810. 공 넣기/공 넣기.js
const rl = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout,
});

let n = 0;
let m = 0;
let baskets = [];

rl.on('line', (line) => {
    if (n === 0) {
        [n, m] = line.split(' ').map(Number);
        baskets = Array(n + 1).fill(0);
    } else {
        const [i, j, k] = line.split(' ').map(Number);

        for (let x = i; x <= j; x++) {
            baskets[x] = k;
        }
        m--;

        if (m === 0) {
            rl.close();
        }
    }
}).on('close', () => {
    for (let i = 1; i <= n; i++) {
        process.stdout.write(`${baskets[i]} `);
    }
    process.exit(0);
});